flow-libs/module.js.flow

```diff
@@ -0,0 +1,84 @@
+//@flow strict-local
+
+// Derived from the documentation and source of Module located at
+// https://nodejs.org/api/module.html#module_modules_module_api and
+// https://github.com/nodejs/node/blob/cbfa2d11d4e42ea7f6a847a6930fd1022773252e/lib/internal/modules/cjs/loader.js/
+// Which is licensed MIT
+
+declare module 'module' {
+  declare type FilePath = string;
+
+  // Derived from https://github.com/facebook/flow/blob/61913b9f0dcd564fd4c74fefbe896b33ef447cab/lib/node.js
+  declare class Url {
```

Comment:
`import type {URL} from 'url'`?

Reply:
looks like this is addressed in the upstream PR :) https://github.com/facebook/flow/pull/8540